source/plugin/permissions.rst

```diff
@@ -370,3 +370,32 @@ The ``ContextCalculator`` can be registered via:
 .. code-block:: java
     
     permissionService.registerContextCalculator(contextCalculator);
+
+For Forge Mods
+========
+
+If you are the author of a Forge mod and are not using the new Forge PermissionsAPI but are doing OP checks then you are already on the right path for Sponge to pick up permissions.
+
+The simplest way to create a Sponge permission in a Forge mod without soft-depending on SpongeAPI is to use the method provided by Vanilla Minecraft code in ``ICommandSender``, namely ``ICommandSender.canCommandSenderUseCommand(int permLevel, String commandName)``. The String passed into that method has no use at all in a Vanilla Forge environment but when Sponge Forge is added it automatically takes that String and converts it into a working permission.
+
+Example
+~~~~~~~
+
+.. code-block:: java
+
+    public Block AwesomeBlock extends Block {
+        @Override
+        public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, EntityPlayer player, EnumFacing side, float hitX, float hitY, float hitZ) {
+            if(player.canCommandSenderUseCommand(4, "examplemod.awesomeblock.interact")) {
+                //Do cool stuff
+                return true;
+            }
+        return false;
+        }
+    }
+
+As you can see, we simply check for the OP level and pass in a random String we want to use as a permission when Sponge is used. When Forge is used by itself the player simply requires the OP level, passing a value of 0 would allow all users to interact with the block but when SpongeForge is added they require the permission node of ``examplemod.awesomeblock.interact``.
+
+.. note::
+    
+    The srg name for this method is ``func_70003_b``
```

Comment:
I'm not sure this will be much use here, as it depends on the MCP mappings, which aren't referenced much in these docs so far, but it could be important for Forge mod devs. I am seeking opinions.

Reply:
The problem with not having it is if they change the name of the method (it used to be hasPermission), plus this is a section about Forge development itself

Reply:
I agree. The name is not dependend on Sponge. Users that already use this kind of OP check already use it. Experienced users will propably use the 'fixed' name anyway. Inexperienced users are unlikely to try to be compatible with Sponges PermSystem. And if they try its probably easy enough to just search for it or even guess it from the parameters.

Reply:
IMO Name changes to the remapped names are less likely than the raw names.

Reply:
Hence the usage, the searge name never changes but the MCP mapping name does

Reply:
If it's fixed for a given SpongeAPI version (which all conveniently happen to belong to different Minecraft versions), then it should be OK. If this PR were to be cherry-picked to earlier API versions, we'd need separate updates.